src/common/constants.ts

```diff
@@ -14,6 +14,10 @@ export const MAX_LIMIT = 100;
 export const NODE_UPTIME_CHECKIN_HOURS = 1; // Check-in can happen once per hour
 export const NODE_UPTIME_CREDIT_HOURS = 12; // Credit in 12 hour chunks
 
+// Send Transaction
+export const ORE_TO_IRON = 100000000;
+export const SEND_TRANSACTION_LIMIT = ORE_TO_IRON * 0.1; // Only transactions >= to this amount will get points
```

Comment:
I think we should use the ORE suffix here, just like how we have _HOURS suffix on time to indicate the time unit